azure-pipelines.yml

```diff
@@ -139,19 +143,28 @@ jobs:
       Python36:
         python.version: '3.6'
         python.org.version: '3.6.8'
+        python.installer.name: 'macosx10.9.pkg'
       Python37:
         python.version: '3.7'
         python.org.version: '3.7.7'
+        python.installer.name: 'macosx10.9.pkg'
       Python38:
         python.version: '3.8'
         python.org.version: '3.8.2'
+        python.installer.name: 'macosx10.9.pkg'
       Python39:
         python.version: '3.9'
         python.org.version: '3.9.0'
+        python.installer.name: 'macosx10.9.pkg'
+      Python310:
+        python.version: '3.10'
+        python.org.version: '3.10.0'
+        python.installer.name: 'macos11.pkg'
```

Comment:
There is no `macosx10.9.pkg` available anymore starting with Python 3.10. The one available is a so-called universal2 installer. The wheel produced is now named: [NEURON_nightly-8.0a710-cp310-cp310-macosx_10_9_universal2.whl](https://dev.azure.com/neuronsimulator/_apis/resources/Containers/18075869/drop?itemPath=drop%2FNEURON_nightly-8.0a710-cp310-cp310-macosx_10_9_universal2.whl). I still need to look into this further, however it apparently should work on both `x86_64` and `arm64`. @nrnhines would you mind trying it out on your M1? I can confirm it works on my Intel Macbook Air:
```python
(venv310) bb-fvfzj1tzlywp:Workspace savulesc$ pip install ~/Downloads/NEURON_nightly-8.0a715-cp310-cp310-macosx_10_9_universal2.whl 
Processing /Users/savulesc/Downloads/NEURON_nightly-8.0a715-cp310-cp310-macosx_10_9_universal2.whl
Collecting numpy>=1.9.3
  Downloading numpy-1.21.5-cp310-cp310-macosx_10_9_x86_64.whl (17.0 MB)
     |████████████████████████████████| 17.0 MB 1.3 MB/s            
Installing collected packages: numpy, NEURON-nightly
Successfully installed NEURON-nightly-8.0a715 numpy-1.21.5
(venv310) bb-fvfzj1tzlywp:Workspace savulesc$ python
Python 3.10.1 (main, Dec  6 2021, 23:20:29) [Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import neuron
>>> neuron.test()
....
----------------------------------------------------------------------
Ran 15 tests in 4.916s

OK
True
>>> neuron.test_rxd()
test_ecs_diffusion_fixed_step (neuron.tests.test_rxd.RxDTestCase) ... ok
test_ecs_diffusion_variable_step_coarse (neuron.tests.test_rxd.RxDTestCase) ... ok
test_ecs_diffusion_variable_step_fine (neuron.tests.test_rxd.RxDTestCase) ... ok
test_rxd (neuron.tests.test_rxd.RxDTestCase) ... ok

----------------------------------------------------------------------
Ran 4 tests in 6.291s

OK
True
>>> quit()

```